src/components/VmDetails/cards/UtilizationCard/DiskCharts.js

```diff
@@ -43,18 +72,20 @@ const DiskCharts = ({ vm, isRunning, id, ...props }) => {
   })
 
   const { unit, value: disks } = convertValueMap('B', { actualSize, provisionedSize })
-  const used = round(disks.actualSize, 1)
   const available = round(disks.provisionedSize - disks.actualSize, 1)
   const total = round(disks.provisionedSize, 1)
 
   return (
     <UtilizationCard className={style['chart-card']} id={id}>
       <CardTitle>Disk <span style={{ fontSize: '55%', verticalAlign: 'super' }}>(storage allocations)</span></CardTitle>
       <CardBody>
-        { !hasDisks &&
-          <NoLiveData id={`${id}-no-live-data`} message='This VM has no attached disks.' />
+        { !diskDetails && isRunning &&
```

Comment:
Humm.... If the guest agent isn't reporting file system use, the disk allocated vs provisioned is still available to render.  I think it makes sense to put the `NoLiveData` messages below the `UtilizationCardDetails`.  The disk allocations, assuming the VM has attached disks, will always be available regardless of the state of the VM.

TL;DR - Put the "no attached disks" `NoLiveData` message check back here and move the 2 new `NoLiveData` messages to the same spot as the `DiskBar`s.